Templates/Tests/Context/ModelMappable.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  ModelMappable.swift
+//  TemplatesTests
+//
+//  Created by Ben Vest on 2/13/18.
+//  Copyright © 2018 Pixle. All rights reserved.
+//
+
+import Foundation
+
+protocol ModelMappable { }
+
+protocol ExampleMapping: ModelMappable {
+  var string: String { get }
+  var bool: Bool { get }
+  // sourcery: coding="some_variable_with_a_long_key"
```

Comment:
I would use `coding_` as common prefix for annotations related to this template, so this annotation would be `coding_key`